profiles/seccomp/seccomp_test.go

```diff
@@ -59,26 +61,6 @@ func TestLoadProfile(t *testing.T) {
 	assert.DeepEqual(t, expected, *p)
 }
 
-func TestLoadProfileWithDefaultErrnoRet(t *testing.T) {
```

Comment:
Wondering why you removed the test here; is it because the one in daemon/ already covers it?

Reply:
No, because now it is tested in the `TestLoadProfile` function here: https://github.com/moby/moby/pull/42649/files#diff-ee871717077bc31d70d119a348d475d268e45ff60583cc67a294103dd79b11f2R29

I can keep the test if you prefer :)

Reply:
Reason I was looking at it was that (but haven't checked all other tests), I felt it was good to keep a test that loads it from the JSON profile (`TestLoadProfile` constructs the config in Go).

Reply:
Ohh, I see. So, I left this test and also left it `TestLoadProfile`. Let me know if you prefer in some other way :)